SimPEG/Regularization.py

```diff
@@ -193,7 +193,19 @@ def cellDiffx(self):
         :return: differencing matrix for active cells in the x-direction
         """
         if getattr(self, '_cellDiffx', None) is None:
-            self._cellDiffx = self._Pafx.T * self.mesh.cellGradx * self._Pac
+            if getattr(self.mesh, 'cellGradx', None) is not None:
+                cellDiffx = self.mesh.cellGradx
+            elif getattr(self.mesh, 'faceDivx', None) is not None:
```

Comment:
@rowanc1 : do you have thoughts on a more elegant implementation for this (getting the interior cells and dealing with hanging faces)? This is a bit of a trick, but does do the job